_inc/client/components/welcome-new-plan/wordads-prompt.jsx

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Are we using this component anywhere? I didn't notice it in the screenshots and when I search `wordads-prompt` on this page, nothing shows up.